webapp/settings.py

```diff
@@ -150,3 +153,5 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+DJANGO_BASE_URL = str(os.getenv('DJANGO_BASE_URL'))
+CSRF_TRUSTED_ORIGINS = [DJANGO_BASE_URL]
```

Comment:
O dynaconf tá configurado aqui: https://github.com/gogixweb/bahea-cal/blob/main/config.py#L5
Como ele tem prefixo `DJANGO_`, a variável fica só 'BASE_URL'.

Já tem um `BASE_URL` na linha 31 também: https://github.com/gogixweb/bahea-cal/blob/main/webapp/settings.py#L31

Então é só fazer:
```suggestion
CSRF_TRUSTED_ORIGINS = [BASE_URL]
```